contracts/utils/Ownable.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity ^0.5.5;
```

Comment:
[Minor] Was it this intended? As far as I know using `^0.5.5` or `^0.5.0` means exactly the same